### ISSUE TITLE
rocket fall damage

### ISSUE THREAD

**Issue by PhysicalNova** [micdoodle8/Galacticraft#3495]
REVIEW THE GUIDELINES (LINK AT TOP OF THIS PAGE)
The tier 2 & 3 rockets make the player take fall damage when he get out. Is it an intended behavior ?
Minecraft version: 1.12.2
Galacticraft version: 1.12.2-4.0.1.181

Single player (SSP)

Galacticraft add-ons and other mods installed? No.

**Comment by TheQuadShot** [comment]
No, not really, How far are you falling? Can you provide more information, maybe screenshots?

**Comment by PhysicalNova** [comment]
From what I've experienced, you take x heart of damage. The player is teleported right above the rocket, making him/her take 1 and 1/2 heart of damage. Here I have a damage indicator. (No mods influence damage taking, so I consider that I'm running only Galacticraft.)
![2018-08-07_20 01 59](https://user-images.githubusercontent.com/40638280/43793912-2361916a-9a7d-11e8-8489-41b4b2e0c555.png)

And here you can see the player above the rocket.
![2018-08-07_20 02 26](https://user-images.githubusercontent.com/40638280/43793962-4a824c08-9a7d-11e8-8064-69115f7ba0ee.png)
So, maybe it's because of some coordinates and math bugs, I don't know.

**Comment by PhysicalNova** [comment]
Then, if it's 3 damages, It's 6 blocks fallen.
